tasks/us/census/acs.py

```diff
@@ -35,6 +35,7 @@
                CBSA, PLACE]
 YEARS = ['2010', '2014', '2015', '2016']
 SAMPLES = [SAMPLE_5YR, SAMPLE_1YR]
+MINIMUM_TIGER_YEAR = 2015
```

Comment:
_Really minor_: maybe there might be a `YEARS` array at `tiger.py` and this could get the first one. that way it'd would also document the available years.

Reply:
That's a nice idea. The years belong to tiger indeed.